components/user/UserProfile.js

```diff
@@ -68,7 +75,16 @@ function UserProfile({ BASE_URL, data }) {
 
       <div className="flex justify-center my-4">
```

Comment:
```suggestion
      <article className="flex justify-center my-4">
```

Reply:
I don't think using the `article` tag here is correct 

> The <article> tag specifies independent, self-contained content.
> An article should make sense on its own and it should be possible to distribute it independently from the rest of the site.
> Potential sources for the <article> element:
> - Forum post
> - Blog post
> - News story

https://www.w3schools.com/tags/tag_article.asp

Reply:
> I don't think using the `article` tag here is correct
> 
> > The  tag specifies independent, self-contained content.
> > An article should make sense on its own and it should be possible to distribute it independently from the rest of the site.
> > Potential sources for the  element:
> > 
> > * Forum post
> > * Blog post
> > * News story
> 
> https://www.w3schools.com/tags/tag_article.asp

@CBID2 I agree with Eddie, div is more than enough for this specific task